From 1fb0ad608b52ce8ccc8fc8766f60fdcd1c28de34 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 21 Nov 2018 16:31:14 +0200
Subject: [PATCH 5176/5242] MLK-20328-4: ARM64: dts: mx8mm-evk: adjust format,
 add 2 pins masks

commit  3caa36e0537687b73e15778d0b32e994756ef098 from
https://source.codeaurora.org/external/imx/linux-imx.git

a) Default value must be prefixed by "0".
b) Map 2 pins case to rx=0xff tx=0x11.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit bb1d5eac8b6ba4371ec04f362071c06cd3f5066b)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index 8042923..b548f4d 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -850,7 +850,7 @@
 		<&clk IMX8MM_AUDIO_PLL2_OUT>;
 	clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3", "pll8k", "pll11k";
 	fsl,sai-multi-lane;
-	fsl,dataline,dsd = <0xff 0x11>;
+	fsl,dataline,dsd = <0 0xff 0xff 2 0xff 0x11>;
 	dmas = <&sdma2 0 26 0>, <&sdma2 1 26 0>;
 	status = "okay";
 };
-- 
1.7.9.5

